### DISCUSSION TITLE
Idea/Feature Request: Horizontal scroller/slider for upcoming shows

### DISCUSSION THREAD

**Discussion by emar** [netmix/radio-station#443] [Ideas]
Hi,

In relation to the same topic on WordPress, 

I like the vertical display for upcoming shows.
But is there any way we can create a horizontal display, with slide arrows.
Or maybe something that slides horizontally by itself, for upcoming shows.

Something like this https://snipboard.io/nGeEU1.jpg

Thanks

**Comment by tonyzeoli** [comment]
I know this is a very old post, but for legacy archive purposes, I'll answer it to say that there are many sliders out that use custom posts to pull the featured image, post title, post date, post excerpt, etc. There's the Essential Grid plugin that does this and I'm sure there are many more - especially with todays page builders. Back when this was asked it probably did exist but people might not want to purchase third party add-ons, so we'll look into this to make it part of Radio Station PRO visual views.
